xknx/core/state_updater.py

```diff
@@ -21,6 +19,7 @@ def __init__(self, xknx):
         self.xknx = xknx
         self.started = False
         self._workers = {}
+        self._one_by_one = asyncio.Lock()
```

Comment:
Maybe renaming this to something like `self._mutex` or `self._lock` would be better readable.

Reply:
Sure. What about `self._mutex_lock`?

Reply:
It should be either lock or mutex. A mutex is basically a system wide lock (over multiple processes). I guess in python lock would be the correct wording

Reply:
I'll test this feature branch and give feedback after :+1:

Reply:
I liked the line "async with one_by_one"    😉

Reply:
Yea also true. Can we use an `asyncio.Semaphore(3)` to update 3 states at the same time? I understand we do not want to spam the bus with all the state updates at once but doing one by one looks kind of unneccessary for me.

Reply:
Sure, should work. I'll try it later.